src/infinity-menu.js

```diff
@@ -266,7 +278,7 @@ export default class InfinityMenu extends React.Component {
 	 *  @description React render method for creating infinity menu
 	 */
 	render() {
-		const tree = dcopy(this.props.tree);
+		const tree = this.props.tree;
```

Comment:
what if the user modifies the tree on their end without searching through the component? just not supported ?